src/foam/dao/history/HistoryRecordServiceServer.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dao.history',
+  name: 'HistoryRecordServiceServer',
+  implements: [ 'foam.dao.history.HistoryRecordService' ],
+
+  javaImports: [
+    'foam.core.X',
+    'foam.dao.ArraySink',
+    'foam.dao.DAO',
+    'foam.mlang.order.Desc',
+    'java.util.stream.Collectors',
+    'java.util.List'
+  ],
+
+  methods: [
+    {
+      name: 'getRecord',
+      type: 'HistoryRecord',
+      args: 'Context x, foam.dao.DAO dao, String propertyName',
```

Comment:
Shouldnt propertyName be used in the method?

Reply:
Thanks for catching it, updated.